src/main/java/com/impactupgrade/nucleus/service/logic/MessagingService.java

```diff
@@ -108,4 +108,23 @@ public CrmContact processContact(
 
     return crmContact;
   }
+
+  public void processSmsOpt(
```

Comment:
@brmeyer Haven't actually tested to see if it updates the contact appropriately in the specified CRM...could use some help on that.